components/resources/quizzes/careerQuiz/career-quiz.tsx

```diff
@@ -50,7 +55,7 @@ const CareerQuiz = () => {
         };
       }),
       currentQuestion: 0,
-      progress: 0,
+      progress: quizData.questions.length,
```

Comment:
progress should be a percent between 0 and 100 that you can just pass into the progress bar. Do the calculations in the state management, not in the progress bar view.